src/components/AuthenticatedAppHeader/NotificationsPane.jsx

```diff
@@ -109,6 +118,24 @@ export default function NotificationsPane({
                     id="VIEW"
                   />
                 </Grid>
+                <Grid
+                  item
+                  style={{
+                    display: 'flex',
+                    paddingLeft: 12,
+                    marginLeft: 4,
+                    width: 'max-content',
+                  }}
+                >
+                  <Text
+                    style={{ color: theme.palette.text.secondary }}
+                  >
+                    {timeSince}{' '}
+                    {intl.formatMessage({
+                      id: 'AGO',
+                    })}
```

Comment:
so we should put the whole phrase in `en.json` using `timeSince` as a value. otherwise this stuff can come out really weird in translation (for example, what about a language where instead of saying "two hours ago" they say "since two hours" // "desde dos horas")